build.rs

```diff
@@ -8,6 +10,19 @@ fn main() {
     } else if env::var_os("CARGO_FEATURE_I128").is_some() {
         panic!("i128 support was not detected!");
     }
+
+    match rustc_version::version_meta() {
+        Ok(ref meta) if meta.semver.major >= 1 && meta.semver.minor >= 32 => {
+            println!("cargo:rustc-cfg=int_to_from_bytes");
```

Comment:
I would rather use another `probe` here, instead of a version check.

I've also been meaning to move num's build-script probes to use the `autocfg` crate (which I also wrote), but you don't have to change that unless you're interested.

Reply:
Interesting, maybe I can submit another PR to `autocfg` for method detection later.